my_list.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'my_enumerable'
```

Comment:
- [ ] It would be better if we change the relative path to `require_relative './my_enumerable'`. I came across error trying to import the module

Reply:
OK.